src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpPrincipal.java

```diff
@@ -63,41 +68,47 @@ public boolean equals (Object another) {
     }
 
     /**
-     * returns the contents of this principal in the form
-     * <i>realm:username</i>
+     * Returns the contents of this principal in the form
+     * <i>realm:username</i>.
+     *
+     * @return the contents of this principal in the form realm:username
      */
     public String getName() {
         return username;
     }
 
     /**
-     * returns the username this object was created with.
+     * Returns the {@code username} this object was created with.
      *
-     * @return The name of the user assoicated with this object
+     * @return the name of the user associated with this object
      */
     public String getUsername() {
         return username;
     }
 
     /**
-     * returns the realm this object was created with.
+     * Returns the {@code realm} this object was created with.
      *
-     * @return The realm associated with this object
+     * @return the realm associated with this object
      */
     public String getRealm() {
         return realm;
     }
 
     /**
-     * returns a hashcode for this HttpPrincipal. This is calculated
-     * as <code>(getUsername()+getRealm().hashCode()</code>
+     * Returns a hashcode for this {@code HttpPrincipal}. This is calculated
+     * as <code>(getUsername()+getRealm().hashCode()</code>.
```

Comment:
There is a close parenthesis missing after `getRealm()`

Reply:
Also `<code></code>` could be replaced with `{@code }`

Reply:
Closing parenthesis added and code tag changed in commit: d8b93d01021d696f9f5272b1e6b34750dc36bcbf